src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1100,6 +1100,9 @@ private void buildDockerImage(File tempDockerfile, File userDockerfile, boolean
             sb.append("-f " + tempDockerfile + " -t " + imageName + " " + userDockerfile.getParent());
             String buildCmd = sb.toString();
             info(buildCmd);
+            if (hasFeaturesSh.get()) {
+                warn("The RUN features.sh command is detected in the Dockerfile and extra time may be necessary when installing features.");
```

Comment:
I think we should change this to log as `info` to be less alarming, and since we don't know if installing features will actually take a long time.

Reply:
Ok!